docs/Attack_Narrative_Template.md

```diff
@@ -0,0 +1,43 @@
+# Attack Narrative - The Template
+The main goal of this document is to describe how a malicious user could exploit a vulnerability, intentionally installed on `your app's name` from secDevLabs, to `add here the vulnerability`.
```

Comment:
`a malicious user could exploit a vulnerability` into `a malicious user could exploit a %NAME_OF_THE_OWASP_TOP_10_VULN% vulnerability`

`from secDevLabs.`

Reply:
Thanks @rafaveira3 ! It does look better!